src/main/java/com/ustudent/resquod/service/AttendanceListService.java

```diff
@@ -1,22 +1,21 @@
 package com.ustudent.resquod.service;
 
 
-import com.ustudent.resquod.exception.EmailExistException;
-import com.ustudent.resquod.exception.EventNotFoundException;
-import com.ustudent.resquod.exception.PermissionDeniedException;
-import com.ustudent.resquod.exception.WrongTimeFrameException;
+import com.ustudent.resquod.exception.*;
 import com.ustudent.resquod.model.AttendanceList;
 import com.ustudent.resquod.model.Event;
+import com.ustudent.resquod.model.Room;
 import com.ustudent.resquod.model.User;
-import com.ustudent.resquod.model.dao.AttendanceListData;
-import com.ustudent.resquod.model.dao.UserData;
+import com.ustudent.resquod.model.dao.*;
```

Comment:
our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.